src/main/java/com/blockout/views/ScrollingContainer.java

```diff
@@ -3,6 +3,7 @@
 import com.blockout.Pane;
 import com.blockout.PaneParams;
 import com.blockout.View;
+import net.minecraft.client.renderer.GlStateManager;
 import org.lwjgl.opengl.GL11;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'org.lwjgl.opengl.GL11'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUselessImportCheck)